Tests/ActionTests/ActionTests.swift

```diff
@@ -354,37 +367,41 @@ class ActionTests: QuickSpec {
 			describe("disabled action") {
 				sharedExamples("send notEnabled errors to errors observable") {
 					it("inputs subject receives generated inputs") {
-						XCTAssertEqual(inputs.events, [
-							Recorded.next(10, "a"),
-							Recorded.next(20, "b")
-							])
+						XCTAssertEqual(inputs.events, Recorded.events([
+                            .next(10, "a"),
```

Comment:
It's still misaligned IMO... Can you make sure all of the events are indented after the `XCT` portion?

Reply:
yeah, for sure. I thought crtl + i on Xcode would solve it all, but apparently not haha

Reply:
Now, I notice that is the default spacing of xcode, should we change it then? or just align this part?

Reply:
Default Xcode alignment is crap :P Let's do the one that makes sense (after XCT)_

Reply:
haha Sure :) Done